### ISSUE TITLE
recover phantom

### ISSUE THREAD

**Issue by SWeinmueller** [MRsources/MRzero-Core#23]
Current version of recover function can't handle complex B1 maps. 
https://github.com/MRsources/MRzero-Core/blob/eca8d705e870dce983ecf5e6ac05eadb061b6668/python/MRzeroCore/phantom/voxel_grid_phantom.py#L469

Change to full = torch.zeros([sparse.shape[0], *mask.shape], dtype = sparse.dtype)

**Comment by J-Endres** [comment]
Fixed by yourself, closing this b701b29eb85f394f895e4fb22973f9d25b5e7999
